dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ElasticsearchResponseException.java

```diff
@@ -0,0 +1,19 @@
+package com.dotcms.content.elasticsearch.business;
+
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+
+/**
+ * Throw when a Elasticsearch request fail
+ */
+public class ElasticsearchResponseException extends Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)